client-config/src/main/java/com/palantir/conjure/java/client/config/ClientConfigurations.java

```diff
@@ -77,6 +78,7 @@ public static ClientConfiguration of(ServiceConfiguration config) {
                 .backoffSlotSize(config.backoffSlotSize().orElse(DEFAULT_BACKOFF_SLOT_SIZE))
                 .clientQoS(CLIENT_QOS_DEFAULT)
                 .serverQoS(PROPAGATE_QOS_DEFAULT)
+                .taggedMetricRegistry(DefaultTaggedMetricRegistry.getDefault())
```

Comment:
May want to upgrade and use the non-deprecated method

Reply:
Aha so I actually don't want to impose tritium 0.12.0 on everyone just yet because it changes the global singleton from EDR -> sliding window, and the intention of this PR is to let us roll out that change in a well communicated way!